python/tvm/micro/contrib/zephyr.py

```diff
@@ -204,6 +204,22 @@ def library(self, output, sources, options=None):
         )
         return tvm.micro.MicroLibrary(build_dir, [f"lib{project_name}.a"])
 
+    def _print_make_statistics(self, output):
+        print("==================")
```

Comment:
want to print this after you see "Memory region"?

Reply:
sure! Now the output looks like:
```
INFO:tvm.micro.contrib.zephyr:Memory region         Used Size  Region Size  %age Used
INFO:tvm.micro.contrib.zephyr:--------------------- ---------- ------------ ---------
INFO:tvm.micro.contrib.zephyr:           FLASH:       35748 B         1 MB      3.41%
INFO:tvm.micro.contrib.zephyr:            DTCM:          0 GB        64 KB      0.00%
INFO:tvm.micro.contrib.zephyr:            SRAM:      146160 B       256 KB     55.76%
INFO:tvm.micro.contrib.zephyr:        IDT_LIST:         232 B         2 KB     11.33%
```